rclpy/rclpy/time_source.py

```diff
@@ -91,12 +97,14 @@ def attach_node(self, node):
 
     def detach_node(self):
         # Remove the subscription to the clock topic.
+        error = RuntimeError('Unable to destroy previously created clock subscription')
```

Comment:
Since this temporary variable is only used in a single case I would rather leave the `RuntimeError` there and not at a variable for it.

Reply:
I was planning to use the `error` more than once, but then I didn't.